lib/action_subscriber/bunny/subscriber.rb

```diff
@@ -17,6 +17,9 @@ def auto_pop!
         times_to_pop = [::ActionSubscriber::Threadpool.ready_size, ::ActionSubscriber.config.times_to_pop].min
         times_to_pop.times do
           queues.each do |route, queue|
+            # Handle busy checks on a per threadpool basis
+            next if route.threadpool.busy_size >= route.threadpool.pool_size
```

Comment:
This is currently a `Lifeguard::InfiniteThreadpool` so there's not much we can do. I plan to extend this in a separate PR and add `#busy?` as an instance method (as well as a threadpool name.. mentioned that above).